python/pynessie/model.py

```diff
@@ -68,7 +68,7 @@ def requires_expected_state() -> bool:
 
     def pretty_print(self) -> str:
         """Print out for cli."""
-        pass
+        raise NotImplementedError
```

Comment:
see `Safe Handling of Empty Function Bodies`
https://mypy-lang.blogspot.com/2022/11/mypy-0990-released.html